spec/helpers/date_helper_spec.rb

```diff
@@ -14,14 +14,18 @@
     context "with valid dates" do
       %w(1/1/2014 3/31/2014 10/2/2014 12/31/2014).each do |date_string|
         it "considers '#{date_string}' a valid date" do
-          expect(parse_usa_import_date(date_string))
-            .to eq DateTime.strptime(date_string, "%m/%d/%Y").to_time_in_current_zone
+          expect(parse_usa_import_date(date_string)).to be_present
```

Comment:
The old test was essentially doing the same parsing as the method it was testing. I've added tests below to check that the day, month, and years are what we expect.